README.md

```diff
@@ -53,6 +53,56 @@ These workflows describe how this information is used.
 
 ### Dialect Testing
 
+#### Local environment
+Testing the dialects locally will require different frameworks/libraries. Following steps
+mentions reference methods:
+- **cuDF**  
+   For testing functions in cuDF, we need a GPU powered system having RAPIDS/cuDF installed.
```

Comment:
I prefer to write documention avoiding referencing any person.  So instead of "we need" I might just say that the RAPIDS/cuDF system needs to be installed.  And elsewhere.

Reply:
sounds right, made the change, thanks.